cgi-bin/LJ/Protocol.pm

```diff
@@ -2024,6 +2027,7 @@ sub editevent {
     }
 
     ## handle meta-data (properties)
+    # FIXME: Hey... I think this just throws the changed props away???? -NF
```

Comment:
Not all props are "user editable" through the protocol, is that what you are referring to?